pkg/cvo/internal/operatorstatus.go

```diff
@@ -97,6 +99,13 @@ func (b *clusterOperatorBuilder) Do(ctx context.Context) error {
 	if b.modifier != nil {
 		b.modifier(os)
 	}
+	if b.mode == resourcebuilder.PrecreatingMode {
+		_, err := b.createClient.Create(os)
+		if err != nil && kerrors.IsAlreadyExists(err) {
```

Comment:
I think we can skip any error.

Reply:
I just want to log unexpected errors in case we find out that early fill is hanging or something